### PR TITLE
introduce Account Acquisition

### PR DESCRIPTION
**Pull request by pshastov**

### CODE REVIEW

**Comment by pshastov** [comment]
It seems **currency** parameter is actually mandatory.

File: Library/AccountAcquisition.cs [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        public int? CostInCents { get; set; }
+        public string Currency { get; set; }
+        public AccountAcquisitionChannel? Channel { get; set; }
+        public string SubChannel { get; set; }
+        public string Campaign { get; set; }
+
+        private const string UrlPrefix = "/accounts/";
+        private const string UrlPostfix = "/acquisition";
+
+        public AccountAcquisition(string accountCode) : this()
```
Comment (bhelx) [inline Library/AccountAcquisition.cs]: Perhaps we want to add Currency to these since it's mandatory?

Comment (pshastov) [reply Library/AccountAcquisition.cs]: According to the docs, Currency is optional. https://dev.recurly.com/docs/create-account-acquisition
While testing I've run into the issue if Currency not set. Currently, I can't reproduce it.
I've modified CreateAccountAcquisitionWithMandatoryParameters test.

**Comment by bhelx** [comment]
@pshastov Is this working for you and ready to go? I can test and merge if so. Code looks fine.

**Comment by pshastov** [comment]
Yes, it is ready to go

**Comment by pshastov** [comment]
No problem. Thanks for an awesome product

**Pull request merged**